poetro_app/lib/features/poetry/domain/repository/poetry_repository.dart

```diff
@@ -13,6 +13,7 @@ abstract interface class IPoetryRepository {
 
   Future<ApiResponse<List<PoetryEntity>>> getPoetryListByKeyword(
```

Comment:
i think  PoetryEntity is not good variable name Poetry is betteer,  getPoetryListByKeyword in poetry repo can not get other entity this means you can change it getAllByKeyword

Reply:
what is poetry?